src/intel/vulkan/anv_extensions.py

```diff
@@ -123,7 +123,7 @@ def __init__(self, version, enable):
     Extension('VK_EXT_debug_report',                      8, True),
     Extension('VK_EXT_depth_clip_enable',                 1, True),
     Extension('VK_EXT_descriptor_indexing',               2,
-              'device->has_a64_buffer_access && device->has_bindless_images'),
+              'device->has_a64_buffer_access && device->has_bindless_images && !ANDROID'),
```

Comment:
This will disable it from all devices, not just APL. I think you could make something like this: !(ANDROID && device->info.is_broxton) so that it would be only for APL.

Reply:
Having said that, I don't think this is a good approach in longer term. Google CDD for Android 10 strongly recommends Vulkan 1.1 support and I would not be surprised that they want to use any available extensions too.

Reply:
Yes, a good news is that upstream mesa has optimization for timeout issue. Some timeout issue has gone with latest code. Will keep monitor the latest code, if this issue also could be fixed with later mesa changes, we will cherry pick fix to our github.